src/CMakeLists.txt

```diff
@@ -134,3 +134,22 @@ install(
     ARCHIVE DESTINATION lib/static COMPONENT applications
     INCLUDES DESTINATION include COMPONENT applications
 )
+
+# Configure appimagetool.AppImage component
+install(
+    TARGETS appimagetool AppRun
+    RUNTIME DESTINATION bin COMPONENT appimagetool.AppImage EXCLUDE_FROM_ALL
+)
+
+find_library(LIBFFI_PATH NAMES ffi)
```

Comment:
As said before, I think it's a bad idea to install external binaries using `make install` etc. That will break packaging etc. (it's not like we would want to support that, but we shouldn't break the few distros which actually do).

Reply:
This command does the same than the `build-appimage.sh` script. The lib will be only installed if the `appimagetool.AppImage` component is requested, otherwise, it will be excluded. So regular packaging is not affected.

Reply:
Still, separation of concerns sort of forbids that kind of stuff.

Reply:
The purpose of this PR is to move as much as possible the logic from the `build-appimage.sh` script to CMake code. Deploying additional dependencies is part of that logic, so it must be done.
 
Those external libs and binaries will not get installed with a regular `make install` see the `EXCLUDE_FROM_ALL` at the end of the install command 
https://github.com/AppImage/AppImageKit/blob/f29eeb9829311cd97b6e5150342fb0d011a2e657/src/CMakeLists.txt#L148_L150

Reply:
I am not willing to merge this in this state...

Reply:
Moving that part back to the bash script.
... did you remember that was you who requested this change?

Reply:
@azubieta I already explained a few times that this is not what I wanted. I think I explained to you that only desktop file, icon etc. should be deployed through CMake. I never wanted to deploy files from outside the repository using our build system, because that's out of scope of a build system IMO.

Reply:
Thanks for moving it back there.